ideascube/mediacenter/management/commands/import_medias.py

```diff
@@ -94,7 +94,7 @@ def add(self, metadata):
             return
 
         path = os.path.join(self.ROOT, original)
-        if not os.path.exists(path):
+        if not os.path.isfile(path):
```

Comment:
This is indeed better than what we had. However, it is also racy.

It would be better to not even check for the file, and just handle opening failures.